tests/serial/watch-integral/WatchIntegralScaleAndNoScale.cpp

```diff
@@ -76,26 +76,26 @@ BOOST_AUTO_TEST_CASE(WatchIntegralScaleAndNoScale)
     // Initialize the mesh
     double dt = interface.initialize();
 
-    int    dataAID = interface.getDataID("DataTwo", meshTwoID);
+    int    dataOneID = interface.getDataID("DataOne", meshTwoID);
     double valueA, valueB, valueC;
 
     while (interface.isCouplingOngoing()) {
 
-      interface.readScalarData(dataAID, idA, valueA);
-      interface.readScalarData(dataAID, idB, valueB);
-      interface.readScalarData(dataAID, idC, valueC);
+      interface.readScalarData(dataOneID, idA, valueA);
+      interface.readScalarData(dataOneID, idB, valueB);
+      interface.readScalarData(dataOneID, idC, valueC);
 
       dt = interface.advance(dt);
     }
     interface.finalize();
 
     {
       std::string fileName = "precice-SolverTwo-watchintegral-WatchIntegral.log";
-      auto        result   = readDoublesFromTXTFile(fileName, 4);
+      auto        result   = readDoublesFromTXTFile(fileName, 3);
       auto        expected = std::vector<double>{
-          1.0, 9.5, 0.0, 3.0,
-          2.0, 12.5, 0.0, 3.0,
-          3.0, 12.5, 0.0, 3.0};
+          1.0, 9.5, 3.0,
+          2.0, 12.5, 3.0,
+          3.0, 0.0, 3.0};
```

Comment:
Took these numbers from `precice-SolverTwo-watchintegral-WatchIntegral.log`. Cannot judge whether this is correct or not.